pkg/resource/deploy/import_test.go

```diff
@@ -211,3 +211,76 @@ func TestImporter(t *testing.T) {
 		})
 	})
 }
+
+func TestImporterParameterizedProvider(t *testing.T) {
+	t.Parallel()
+	t.Run("ensure provider is called correctly with Parameterization", func(t *testing.T) {
```

Comment:
Don't think you need this run? Can always put the name in a doc comment for the test if that was the idea?

Reply:
Ack, copied this from the other PR and didn't think about it